build-image.sh

```diff
@@ -25,31 +24,30 @@ image_tag="${DOCKER_IMAGE_TAG:-$graalvm_version-java$java_major_version}" # e.g.
 #################################################
 case $graalvm_version in
    dev)   graalvm_version=$(curl -fsSL -H "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/graalvm/graalvm-ce-dev-builds/releases/latest | grep "tag_name" | cut -d'"' -f4)
-          graalvm_url="https://github.com/graalvm/graalvm-ce-dev-builds/releases/download/${graalvm_version}/graalvm-community-java${java_major_version}-linux-amd64-dev.tar.gz"
+          graalvm_url="https://github.com/graalvm/graalvm-ce-dev-builds/releases/download/${graalvm_version}/graalvm-community-java${java_major_version}-linux-ARCH-dev.tar.gz"
           ;;
 
-   *dev*) graalvm_url="https://github.com/graalvm/graalvm-ce-dev-builds/releases/download/${graalvm_version}/graalvm-community-java${java_major_version}-linux-amd64-dev.tar.gz"
+   *dev*) graalvm_url="https://github.com/graalvm/graalvm-ce-dev-builds/releases/download/${graalvm_version}/graalvm-community-java${java_major_version}-linux-ARCH-dev.tar.gz"
           ;;
 
    latest) case $java_major_version in
               11) graalvm_version="22.3.3";
-                  graalvm_url="https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-${graalvm_version}/graalvm-ce-java11-linux-amd64-${graalvm_version}.tar.gz"
+                  graalvm_url="https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-${graalvm_version}/graalvm-ce-java11-linux-ARCH-${graalvm_version}.tar.gz"
                   ;;
               *)  graalvm_version=$(curl -sSfL -N https://github.com/graalvm/graalvm-ce-builds/tags | grep -oP "releases/tag/jdk-\K${java_major_version}.[^=\"]+" | head -n1)
-                  graalvm_url="https://github.com/graalvm/graalvm-ce-builds/releases/download/jdk-${graalvm_version}/graalvm-community-jdk-${graalvm_version}_linux-x64_bin.tar.gz"
+                  graalvm_url="https://github.com/graalvm/graalvm-ce-builds/releases/download/jdk-${graalvm_version}/graalvm-community-jdk-${graalvm_version}_linux-ARCH_2_bin.tar.gz"
```

Comment:
It would be nice to have a small comment about the difference between ARCH and ARCH_2 before the first case statement here.